itou/users/models.py

```diff
@@ -322,31 +316,51 @@ def save(self, *args, **kwargs):
         self.department = department_from_postcode(self.post_code)
         self.validate_unique()
 
-        # TODO(rsebille): Replace by a UniqueConstraint once the data have been cleaned
-        user_kinds = [
-            self.is_job_seeker,
-            self.is_prescriber,
-            self.is_siae_staff,
-            self.is_labor_inspector,
-        ]
-        if user_kinds.count(True) > 1:
-            raise ValidationError("A User can not have more than one kind")
+        # Update is_staff with kind
+        if self.kind == UserKind.DJANGO_STAFF:
+            self.is_staff = True
+        elif self.kind in UserKind:
+            # Any other user kind isn't staff
+            self.is_staff = False
+        else:  # self.kind == "" : handle django admin_client pytest fixture
+            if self.is_staff:
+                self.kind = UserKind.DJANGO_STAFF
+            else:
+                ValidationError("User is missing kind")
 
         super().save(*args, **kwargs)
 
+    # FIXME(alaurent) keep for templates for now
```

Comment:
Pourquoi `for now`. Ca resterais AMA plus lisible dans un template qu'un test d'égalité et ça mange pas de pain.

Reply:
C'était un commentaire le temps de décider comment je traitais la question.